admin-dev/themes/default/template/controllers/customer_threads/helpers/view/view.tpl

```diff
@@ -80,13 +80,9 @@
 						</div>
 					{/if}
 				</div>
-				<div class="row">
-					<div class="col-sm-12">
-						{if !$first_message.id_employee}
-							{include file="./message.tpl" message=$first_message initial=true}
-						{/if}
-					</div>
-				</div>
+				{if !$first_message.id_employee}
+					{include file="./message.tpl" message=$first_message initial=true}
+				{/if}
```

Comment:
The previous row/col is closed, so the message content doesn't require its ow row/col ?

Reply:
No :)